### ISSUE TITLE
How to output number with fixed decimals and with comma instead of period?

### ISSUE THREAD

**Issue by q2apro** [CindyJS/CindyJS#715]
I am outputing some numbers that needs to be rounded to the first decimal. I am using:

     round(num*10)/10;

which gives me `1, 1.1, 1.2 ...`

I need to print the `1` also as `1.0`.

Is there an elegant way to do this with CindyJS? In JS it is `tofixed()`.

<br>
Furthermore, is there a way to output `4,1` instead of `4.1`?

**Comment by montaga** [comment]
In my opinion, the nicest would be to have a modifier for the `format` command (https://cindyjs.org/ref/String_Operators.html#format$2) that just does this.

As a workaround, you could build your own `format` function similar to the function `fmt` in https://cindyjs.org/examples/81_Clock.html

**Comment by q2apro** [comment]
> the nicest would be to have a modifier for the format command 

Yes, that would be great! Like an option you pass when initializing CindyJS. E.g. `decimals: ",",`

**Comment by kortenkamp** [comment]
Sorry to rain on the parade, but I doubt that such a quick fix is good. Better would be proper localisation support.

**Comment by strobelm** [comment]
@kortenkamp is right, I'm currently working on a branch with proper localization. So perhaps one should not rely on the current syntax.

**Comment by q2apro** [comment]
When is the implementation planned? It would be great to have x,y already in production mode :) For now it's still period which might confuse some young students that got used to comma in school.
